protocols/manage/propagate.go

```diff
@@ -121,7 +121,8 @@ func (p *Propagate) Dispatch() error {
 		p.Unlock()
 		select {
 		case msg := <-p.ChannelSD:
-			log.Lvl3(p.ServerIdentity(), "Got data from", msg.ServerIdentity)
+			log.Lvl3(p.ServerIdentity(), "Got data from", msg.ServerIdentity, "and setting timeout to", msg.Msec)
+			p.sd.Msec = msg.Msec
```

Comment:
It did only work if the timeout was OK with 1sec. If a new propagation-protocol has been started with a bigger timeout, all children nodes kept the 1sec. timeout.

So this makes it work as required.